qiskit_quantuminspire/api/settings.py

```diff
@@ -66,3 +71,20 @@ def from_config_file(cls, path: Path = API_SETTINGS_FILE) -> ApiSettings:
 
         api_settings = path.read_text()
         return ApiSettings.model_validate_json(api_settings)
+
+    @staticmethod
+    async def _fetch_team_member_id(host: str, access_token: str) -> int:
```

Comment:
Fetching and storing the team member id is only needed for the e2e test right? If yes, then this probably shouldn't be in the production code.

Reply:
right, removed it

Reply:
I still see the _fetch_team_member_id and get_team_member_id functions in this file